src/transformation.js

```diff
@@ -778,6 +784,19 @@ class Transformation extends TransformationBase {
     return this.param(value, "page", "pg");
   }
 
+  placeholder(value) {
+    const mode = PLACEHOLDER_IMAGE_MODES[value] || PLACEHOLDER_IMAGE_MODES.blur;
+    const width = this.getValue("width");
+    const height = this.getValue("height");
+    let transformations = mode({width, height});
+    this.chain();
+    transformations.forEach(t=>{
+      this.rawTransformation(new Transformation(t));
+      this.chain();
+    });
```

Comment:
Pretty sure that `Transofmration(PLACEHOLDER_IMAGE_MODES["pixelate"])` will work

Reply:
you're right, fixed